sql_app/crud.py

```diff
@@ -20,7 +20,10 @@ def list_movies(db: HashableSession, engine: str, page: int, num: int):
     """
     Get List of Movies. With pagination and a total of `num` rows per query
     """
-    return list(db.query(models.Movie).filter(models.Movie.engine == engine).limit(num).offset(num * (page-1)))
+    return list(db.query(models.Movie)\
+                .filter(
+                    func.lower(models.Movie.engine) == engine.lower())\
```

Comment:
This will incur additional database costs. We should look towards movie all future queries to use lowercases. Then gradually update the rest in the database to lowercase. What do you think?

Reply:
func.lower doesn't incur any additional database costs. It is the python equivalent of running sql LOWER and is optimized to be such. It's like saying running .lower() in python will be expensive. Having multiple dicts in every codebase is just asking for bugs. The optimization issue comes from pagination

Reply:
That's okay then. We could however look into standardizing it to lowercase if it would bring up any further issues

Reply:
Sure